TriforkSwiftExtensions/TriforkSwiftExtensions/Collection/CollectionExtension.swift

```diff
@@ -14,5 +14,10 @@ public extension Collection {
     public subscript (safe index: Index) -> Iterator.Element? {
         return self.indices.contains(index) ? self[index] : nil
     }
+    
+    /// Tells if this collection is NOT empty (contains element(s))
+    public var isNotEmpty: Bool {
```

Comment:
I'm not sure about this as there already is an empty? @tkctrifork any opinions ?

Reply:
I guess it is because `isNotEmpty` is more explicit than `!isEmpty`. It is very easy to miss the `!` when reading the code. I like it 👍

Reply:
@kdvtrifork Yes, as stated in the top part / PR text, this is for readability, and imagine cases where you negate futhere than the "isEmpty"; say
if (!(x || !isEmpty()) || isEmpty())
not only does the esclamation mark (!) dissapear, but also you easily end up with mulitple levels of negation, causing , some very hard to read and write code.
where as
if ( ! (x || isNotEmpty()) || isEmpty()) 
helps :)

Reply:
Point taken! 🚀